SU-SWS/stanford_basic
# D8CORE-4037: fixed word wrapping with very long menu items

# READY FOR REVIEW

# Summary
- In some, but not all, cases, long menu items weren't wrapping correctly when there was a submenu for the item.  This corrects that problem by adjusting the max-width for items that have a submenu below them, and removes the unnecessary width restriction when there is no submenu.

# Review By (Date)
- end of sprint
- 
# Urgency
- low 

# Steps to Test

1. check out branch, clear caches.
2. create some pages that have long titles, and set them up as drop down menus.
3. make several levels if you can.
4. verify that the long menu items wrap correctly both with submenus and without.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
